Makefile

```diff
@@ -0,0 +1,29 @@
+SHELL := /bin/sh
+.PHONY: install-requirements build run restart stop logs
+
+
+install-requirements:
+	@curl -fsSL https://get.docker.com -o get-docker.sh && sh get-docker.sh
+	pip install docker-compose --user
+
+
+build:
+	@cp .example.variables.env .variables.env
```

Comment:
Aqui teremos problema eu acho.. se já tiver um .variables vai sobrescrever

Reply:
@rochacbruno verifiquei na documentação que posso usar o copy com a flag -n assim ele não sobrescreve caso o arquivo já exista.
[http://manpages.ubuntu.com/manpages/precise/en/man1/cp.1.html](http://manpages.ubuntu.com/manpages/precise/en/man1/cp.1.html)

Reply:
faço o merge assim que isso ai for arumado :)

Reply:
corrigido